oracle/Cargo.toml

```diff
@@ -14,21 +14,24 @@ ocaml = { version = "0.22.0", optional = true }
 rand = "0.7.3"
 rayon = { version = "1" }
 
-# for tests
-serde = { version = "1.0", features = ["derive"] }
-serde_json = { version = "1.0" }
-hex = { version = "0.4" }
-
 # for export_test_vectors
 num-bigint = { version = "0.3", optional = true }
 custom_derive = { version = "0.1.7", optional = true }
 enum_derive = { version = "0.1.7", optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_json = { version = "1.0", optional = true }
+hex = { version = "0.4", optional = true }
+
+[dev-dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
+hex = { version = "0.4" }
```

Comment:
you don't need to re-add them here since they are already listed as normal dependencies above